ssc/elections/2023H1/README.md

```diff
@@ -0,0 +1,15 @@
+# 2023 H1 SSC Election
+This subdirectory captures the nominees and results of the 2023 H1 SSC election. The timeline for this election is as follows:
+* April 5th, 2023: Nominations open
+* April 19th, 2023: Nominations close
```

Comment:
I think 21st would be the close date

Reply:
Ok - is the polls open date below accurate? (26th)

Reply:
@dfeldman can you confirm if we are shifting the rest of timelines, or only the nominations?